.github/workflows/nodejs.yml

```diff
@@ -52,7 +52,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        node-version: [12.x, 14.x, 16.x]
+        node-version: [12.x, 14.x, 16.x, 17.x, 18.x]
```

Comment:
Can you add only 18, we don't need `17` here, it is not LTS

Reply:
done